buildpack/util.py

```diff
@@ -237,6 +237,43 @@ def get_mpr_file_from_dir(directory):
         return None
 
 
+def set_up_launch_environment(deps_dir, profile_dir):
+    profile_dirs = get_existing_deps_dirs(deps_dir, "profile.d", deps_dir)
```

Comment:
Should this be `.profile.d` instead of `profile.d`?

Reply:
The directory name in buildpack dependenices is `profile.d`, and it gets copied to `.profile.d` build folder.

From the [docs](https://docs.cloudfoundry.org/buildpacks/custom.html#contract):
> To make use of dependencies provided by previous buildpacks, the last buildpack:
> * May copy files from /profile.d to /tmp/app/.profile.d during staging.